tests/e2e/cmd/setup/basic/init.sh

```diff
@@ -84,7 +86,7 @@ config="
 .app_state.bank.denom_metadata[0].denom_units[3].denom = \"tkx\" |
 .app_state.bank.denom_metadata[0].denom_units[3].exponent = 18
 "
-echo $(jq "$config" tmp/val1/.titand/config/genesis.json) > tmp/val1/.titand/config/genesis.json
+echo "$(jq "$config" tmp/val1/.titand/config/genesis.json)" >tmp/val1/.titand/config/genesis.json
```

Comment:
```suggestion
echo $(jq "$config" tmp/val1/.titand/config/genesis.json) >tmp/val1/.titand/config/genesis.json
```

Reply:
Wrap param string inside "" character can help avoid wrong echo output if string contain space or newline charater.

EX:

file `a.txt` with content:
```
a b
c
```

with command :

```shell
echo $(cat a.txt) > b.txt
```

will lead to `b.txt` content like this:

```
a b c
```

Reply:
Then please add "" for similar echo commands as well